### PR TITLE
enable eslint with prettier formatting in project

### PR DESCRIPTION
**Pull request by DShumaev**
added eslint and prettier in project

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "airbnb-base",
+    "plugin:prettier/recommended",
+    "prettier"
+  ],
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true
+  },
+  "rules": {
```
Comment (evgenybutkevich) [inline .eslintrc]: If object doesn't have any properties (empty object) just `{}` is enough, no need to add empty lines, the same goes for empty blocks

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+node_modules
+.git
+.gitignore
+.idea
```
Comment (evgenybutkevich) [inline .eslintignore]: Please, don't forget about empty lines at the end of the files and `.gitignore` (I remember that you were going to fix this in your later PRs, just a reminder)

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (evgenybutkevich) [inline .eslintrc:1]: You have got `.eslintrc`, but how about config for `prettier`?
Just few useful articles:
1. https://maxpfrontend.ru/vebinary/nastroyka-eslint-prettier-pre-commit-hook-create-react-app-visual-studio-code/
2. https://ru.hexlet.io/blog/posts/rukovodstvo-eslint-prettier

Also, please, pay attention to the `pre-commit hook` in the first article. This is a very useful control mechanism while developing

File: .eslintrc (hunk 1 above)
Comment (DShumaev) [reply .eslintrc]: Okay

File: .eslintrc (hunk 3 above)
Comment (DShumaev) [reply .eslintrc:1]: I think in this case .prettierrc doesn't matter because main rules determine in linter.
pre-commit hook - I will look at it.

**Review by evgenybutkevich — APPROVED** [review]

**Pull request merged**